en/docs/learn/analytics/white-labeling-tenants.md

```diff
@@ -0,0 +1,23 @@
+# White Labeling for Tenants
+This section explains how to white label analytics dashboard for tenants.
+1. Add following configuration in the deployment.yaml
+    ```yaml
+    themeConfigProviderClass: org.wso2.analytics.apim.dashboards.theme.config.provider.CustomDashboardThemeConfigProvider
+    logoFileName: logo.png
+    faviconFileName: favicon.ico
+    ```
+2. Create a tenant in APIM node 
+Log in to the Management Console.
```

Comment:
This step is not needed for white labeling rather testing it. Isn't it better to ask the user to create white labeled folder with images for an existing tenant? WDYT?